hq/app/views/snyk/snykOrgProjects.scala.html

```diff
@@ -20,26 +21,64 @@ <h5>Top 5 vulnerable projects</h5>
             @for(project <- sortProjects(orgIssues.projectIssues).take(5)) {
                 <tr>
                     <td>
-                        <a target="_blank" rel="noopener noreferrer" href="@linkToSnykProject(project, orgIssues.organisation, None)">@project.project.fold("")(_.name)</a>
+                        <a target="_blank" rel="noopener noreferrer" href="@project.project.fold("")(_.url)">@project.project.fold("")(_.name)</a>
                     </td>
                     <td>@project.critical</td>
                     <td>@project.high</td>
                     <td>@project.medium</td>
                     <td>@project.low</td>
-            <tr>
+                </tr>
             }
             </tbody>
         </table>
     </div>
 }
 
+@topOldVulnerabilities() = {
+    @defining(top5OldCriticalOrHighVulnerabilities(orgIssues)) { oldIssues =>
+        @if(oldIssues.nonEmpty) {
+            <div class="row">
+                <div class="col s12">
+                    <h5>Top 5 old critical/high vulnerabilities</h5>
```

Comment:
Minor point, but while the context for what old means here is right at the top ("older than the maximum permitted age"), at first glance I wasn't sure if this meant that it's just sorting them by age. I wonder if it would make it a bit clearer to make explicit that these are the top 5 vulns that are "breaching the SLO" or something to that effect.

Reply:
Yeah, it would be nice to add a detailed explainer to this dashboard like we have for IAM/SGs. Tempted to address separately?

Reply:
Yep, totally!